Mathlib/Analysis/SpecialFunctions/Exp.lean

```diff
@@ -177,6 +177,11 @@ theorem tendsto_exp_atTop : Tendsto exp atTop atTop := by
   exact tendsto_atTop_mono' atTop B A
 #align real.tendsto_exp_at_top Real.tendsto_exp_atTop
 
+lemma _root_.Filter.Tendsto.exp_atTop {f : α → ℝ} (hf : Tendsto f l atTop) :
```

Comment:
I am not sure this one and the next one are really useful; aren't they just `Real.tendsto_exp_atTop.comp` and `Real.tendsto_exp_atBot.comp` (which is barely longer than calling this lemma by its name).

Reply:
Yes, fair enough.